src/components/BatchAllocations/BatchAllocationLogs/BatchAllocationLogs.js

```diff
@@ -0,0 +1,67 @@
+import { useCallback } from 'react';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import { useUsersBatch } from '@folio/stripes-acq-components';
+
+import {
+  LEDGERS_ROUTE,
+  BATCH_ALLOCATIONS_SOURCE,
+} from '../../../common/const';
+import { resolveDefaultBackPathname } from '../utils';
+import { useBatchAllocationLogs } from '../hooks';
+import { BatchAllocationLogsList } from './BatchAllocationLogsList';
+import { useBatchAllocationLogsMutation } from './useBatchAllocationLogsMutation';
+
+export const BatchAllocationLogs = ({
+  history,
+  location,
+  match,
+}) => {
+  const { id: sourceId } = match.params;
+  const sourceType = location.pathname.includes(LEDGERS_ROUTE) ?
+    BATCH_ALLOCATIONS_SOURCE.ledger :
+    BATCH_ALLOCATIONS_SOURCE.group;
+  const backPathname = resolveDefaultBackPathname(sourceType, sourceId);
```

Comment:
It returns the "default" pathname, you should get a user back to the logs with activated filters on this page closure